### PR TITLE
User Guide Updates for WQX 3.0 domain value lists

### PR DESCRIPTION
**Pull request by lstanish-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

User Guide updates
-----------
- add links to missing WQX 3.0 domain value lists in final table
- add All_Domain values list
- minor text formatting updates
- add section on how to cite WQP

Description
-----------
See JIRA tickets:
- [WQP-1780](https://internal.cida.usgs.gov/jira/browse/WQP-1780)
- [WQP-1951](https://internal.cida.usgs.gov/jira/browse/WQP-1951)
- [WQP-1952](https://internal.cida.usgs.gov/jira/browse/WQP-1952)

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by lstanish-usgs** [comment]
@chopkins-usgs @mbucknell Please review/approve!
Still unclear about when the ChangeLog should be updated and by whom?
Thanks.

**Comment by mbucknell** [comment]
For the changelog, when you change something it should be updated with the approriate tag (this link explains what to do: http://keepachangelog.com/en/1.0.0/). Put the new entry under the Unreleased section. When it is time to make a release, I update the changelog by hand so we know what changed in a particular version.

File: server/wqp/markdown/portal_userguide.html [hunk 1]
```diff
@@ -3557,10 +3573,23 @@ <h3><strong>Water Quality Exchange Allowable value lists and
...
+Monitoring Council, United States Geological Survey (USGS),
+Environmental Protection Agency (EPA); 2021. <a href="https://www.waterqualitydata.us" class="uri">https://www.waterqualitydata.us</a>.</li>
+</ul>
+<p>Citations for specific datasets should use this format:</p>
+<ul>
+<li>National Water Quality Monitoring Council, YYYY, Water Quality
+Portal, accessed mm, dd, yyyy, hyperlink_for_query</li>
+</ul>
+</div>
+</div>
```
Comment (mbucknell) [inline server/wqp/markdown/portal_userguide.html:3592]: Is this needed? Seems like there is an extra close tag.

**Review by chopkins-usgs — COMMENTED** [review]
Great work integrating several changes @lstanish-usgs !

**Comment by lstanish-usgs** [comment]
@mbucknell OK, I will add specifics in the CHANGELOG.

**Pull request merged**